### PR TITLE
Update to 21.08 runtime

### LINKED ISSUE #4: Uses an EOL runtime
This uses the Freedesktop SDK 18.08 runtime which is EOL. Please update to a recent runtime.

### PR DESCRIPTION
**Pull request by wjt**
Fixes #4.

The game still crashes in its default configuration (#5) but this can be worked around by disabling decals in settings, and is unrelated to this change.

### CODE REVIEW

**Comment by wjt** [comment]
Alright, I give up for now.

**Comment by Mailaender** [comment]
Try
```patch
-  "rename-appdata-file": "lugaru.appdata.xml",
+  "rename-appdata-file": "lugaru.metainfo.xml",
```

File: io.gitlab.osslugaru.Lugaru.json [hunk 1]
```diff
@@ -1,20 +1,20 @@
 {
   "app-id": "io.gitlab.osslugaru.Lugaru",
   "runtime": "org.freedesktop.Platform",
-  "runtime-version": "18.08",
+!  "runtime-version": "21.08",
```
Comment (Mailaender) [inline io.gitlab.osslugaru.Lugaru.json]:
```suggestion
  "runtime-version": "21.08",
```

**Comment by wjt** [comment]
Thanks @Mailaender !

**Comment by hfiguiere** [comment]
bot, build

**Comment by hfiguiere** [comment]
See #7 that fixes the build.

**Pull request merged**